src/thunderbots/software/ai/primitive/move_primitive.h

```diff
@@ -9,6 +9,13 @@
 #include "geom/angle.h"
 #include "geom/point.h"
 
+enum AutokickFlag
```

Comment:
Can we rename to `AutokickType` since flags are usually binary?

Reply:
Same goes for functions like `getAutokickFlag`

Reply:
good idea. fixed